src/controllers/v1.controller.ts

```diff
@@ -236,15 +237,17 @@ export class V1Controller {
 
       return await this.pocketRelayer.sendRelay(options)
     } catch (e) {
-      logger.log('error', e.message, {
+      const exception: ErrorObject = e
```

Comment:
I think more checks have to be done here, this is a general catch and I don't think all the errors that might through will have the rpc format